genericjpa/db/src/main/java/org/hibernate/search/db/EventType.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.db;
+
+import org.hibernate.search.exception.AssertionFailure;
+
+/**
+ * Contains constants that describe the different Database-Events that are relevant to the index
+ *
+ * @author Martin Braun
+ * @hsearch.experimental
+ */
+public final class EventType {
```

Comment:
Ok, what I really meant to say is: Please make it an enum ;) You can have static and instance methods for these, too.